apps/frontend/src/assets/images/icons/check.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg width="18" height="18" viewBox="0 0 18 18" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I have only one question. Why do we need to replace `fill="none"`?

Reply:
It's just for consistency. Either "currentColor" or "none". I chose currentColor because some icons don't have fill in path tag. Maybe it will be not so confusing as it would be if we had none (and no path).